app/page/template/_dist/app.htm

```diff
@@ -162,6 +162,7 @@
 <script src="/script/conversation/ConversationStateHandler.js"></script>
 <script src="/script/conversation/ConversationStatus.js"></script>
 <script src="/script/conversation/ConversationStatusIcon.js"></script>
+<script src="/script/conversation/ConversationTimedHandler.js"></script>
```

Comment:
Should we stick to the term ephemeral that  we have been using throughout the app.

Reply:
I am trying to slowly move the terminology to 'timed messages' as this is the new name for this feature.

Reply:
It is not the new name. It is the marketing name. This naming could interfere with the other potential things from the roadmap.

Reply:
I think you are right. It's easier and less error prone to stick with 'ephemeral' on the code side.